notifications/NotificationPushsafer.cpp

```diff
@@ -77,22 +77,27 @@ bool CNotificationPushsafer::SendMessageImplementation(
 			PS_t = cSubject;
 		}
 
-		if (HTTPClient::GETBinary(CURLEncode::URLDecode(PS_p), ExtraHeadersBinary, camimage, 10))
+		if (PS_p != "")
```

Comment:
can you change this to

`is (!PS_p.empty())`

Reply:
changed this